src/EnergyPlus/UnitarySystem.cc

```diff
@@ -4315,17 +4248,17 @@ namespace UnitarySystems {
         if (UtilityRoutines::SameString(loc_supFanPlace, "BlowThrough")) this->m_FanPlace = FanPlace::BlowThru;
```

Comment:
Is there no `enum class` defined for this yet?

Reply:
I'll get to this eventually.